packages/components/src/progress-bar/styles.ts

```diff
@@ -21,11 +21,12 @@ const animateProgressBar = keyframes( {
 // Width of the indicator for the indeterminate progress bar
 export const INDETERMINATE_TRACK_WIDTH = 50;
 
-export const Track = styled.div`
+export const Track = styled.div< { hasUnconstrainedWidth?: boolean } >`
 	position: relative;
 	overflow: hidden;
 	width: 100%;
```

Comment:
```suggestion
```

Could we remove this too, maybe? Should default to `auto`, which should behave the same in our case.